actors/common/src/main/java/org/sunbird/learner/util/ContentStoreUtil.java

```diff
@@ -0,0 +1,46 @@
+package org.sunbird.learner.util;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.util.HashMap;
+import java.util.Map;
+import org.apache.http.HttpHeaders;
+import org.sunbird.common.models.util.HttpUtil;
+import org.sunbird.common.models.util.JsonKey;
+import org.sunbird.common.models.util.LoggerEnum;
+import org.sunbird.common.models.util.ProjectLogger;
+import org.sunbird.common.models.util.ProjectUtil;
+
+public class ContentStoreUtil {
```

Comment:
Move this to sunbird-utils/content-util (new project).

content-util/src/main/java/org/sunbird/content/util/ContentStoreUtil.java